Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [profile.release]
-panic = "unwind"
+panic = "abort"
```

Comment:
Not exactly, `panic = "abort"` has some effects on backtrace due to the compiler optimization, but it does not disable it.
Unwinding means propagating the panic up the call stack so that it can be caught by `catch_unwind` and the resource in the call stack could be destroyed usually (Drop::drop could be called).
Aborting means disabling the unwinding, saying it would print the backtrace and abort immediately when panicked.

**Pruntime was set to `panic = "abort"` from the beginning in the `standalone/pruntime/Cargo.toml`, this change is for the `pink.so` because it belongs to the top workspace.**

Reply:
Thanks for the explainatino